composer.json

```diff
@@ -6,35 +6,33 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.2",
-        "drupal/core": "^8.7",
-        "drupal/language_selection_page": "2.3",
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9",
+        "drupal/language_selection_page": "^2.7",
         "drupal/pathauto": "^1.6"
     },
     "require-dev": {
         "composer/installers": "~1.5",
         "consolidation/annotated-command": "^2.8.2",
-        "consolidation/robo": "~1.4",
-        "drupal/core-composer-scaffold": "^8.8",
+        "consolidation/robo": "^1.4.13 || ^2",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
         "drupal/config_devel": "~1.2",
-        "drupal/devel": "~1.2",
+        "drupal/devel": "~4.1",
         "drupal/drupal-extension": "~4.0",
-        "drupal/redirect": "^1.4",
-        "drush/drush": "~9.0@stable",
-        "guzzlehttp/guzzle": "~6.3",
+        "drupal/redirect": "^1.6",
+        "drush/drush": "~10.0",
```

Comment:
Let's require Drush 10.3 and then we don't need `"consolidation/robo": "^1.4.13 || ^2",`. Also, let's try without `consolidation/annotated-command`. And if good, you can remove also the _readme.